tilerator/runserver.sh

```diff
@@ -2,8 +2,8 @@
 set -e
 
 # wait for cassandra to be up
-CASSANDRA_SERVER=cassandra
-CASSANDRA_PORT=9042
+CASSANDRA_SERVER="${CASSANDRA_HOST:-cassandra}"
```

Comment:
A similar environment variable is already defined in the  `tilerator` image 
https://github.com/Qwant/kartotherian_docker/blob/83e79eb530365cadabdc67b621724595f93ecde9/tilerator/Dockerfile#L61

Its value is the hostname used to contact all Cassandra nodes, so it can be used directly: 
```suggestion
CASSANDRA_SERVER="$TILERATOR_CASSANDRA_SERVERS"
```